Makefile

```diff
@@ -194,8 +195,7 @@ deps:
 
 .PHONY: mocks
 mocks:
-	@$(MAKE) deps:mocks && \
-	find . -type d -name "mocks" -exec rm -r {} + && \
+	@$(MAKE) deps:mocks
```

Comment:
todo: Why was this removed? I just added this fix while generating mocks. Please add it back